gdnative_api.json

```diff
@@ -17,7 +17,1153 @@
           "major": 1,
           "minor": 2
         },
-        "next": null,
+        "next": {
+          "type": "CORE",
+          "version": {
+            "major": 1,
+            "minor": 3
+          },
+          "next": null,
```

Comment:
The entire point of having the complexity of versioned API structs and `next` pointers is so that the layout of older versions can be expected to *never* change. I understand that you need to make breaking changes, and I agree that they're for the better, but *please* just erase the previous versions and make it version 2.0. There isn't any value in keeping retroactively changed "previous versions", except making the life of GDNative API consumers harder than it already is.

Reply:
By leaving the major version at `1`, binaries built against older headers will have no idea the struct layout is different. When used, they will just try to carry on and cause arbitrary chaos because the functions no longer do what they are supposed to. This will require terrible hacks in all bindings/plugins to work around, and it's all avoidable with a simple change to `2`. Please do take a moment to consider putting `"major": 2` there. It'll cost a few seconds, nothing will be lost, and the API will be much less of a pain to deal with. Thank you very much.